db/migrate/20160418141210_add_read_only_to_miq_alert.rb

```diff
@@ -1,9 +1,17 @@
+require 'miq_expression.rb'
+
 class AddReadOnlyToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base; end
+
+  def self.guids_from_yaml
+    YAML.load_file("db/fixtures/miq_alerts.yml").map { |h| h["guid"] || h[:guid] }
```

Comment:
> I think the seed code should hard-set read_only and not rely on it being in the yml or not.

Ok great, yeah, that was my first thought.  Honestly, I just wanted to remove the dependence on MiqExpression completely.  

So, I think this is what we want to do:
* MiqAlert.seed setting read_only => true for new rows
* This migration should hardcode a Set of guids from the current yml file so we can migrate any existing rows to read_only

We can then remove the miq_expression require and the YAML.load_file stuff. 🎉